.travis.yml

```diff
@@ -1,5 +1,8 @@
 dist: bionic
 language: python
+stages:
+  - Tests
```

Comment:
Think this `Tests` `stage` needs to be used in the `jobs`:
https://github.com/aws-cloudformation/cfn-python-lint/blob/bb1a9c4875a9207da893d1eb68028606b7ff8350/.travis.yml#L11